Week-5/assignment-2/index.js

```diff
@@ -0,0 +1,25 @@
+function isVowel(char) {
```

Comment:
Do we really need this func? The whole purpose is to improve the reference example. Think in that direction

Reply:
also read the guidelines again

Reply:
- They have themselves given the above isVowel function
- we need to discuss this as well on next 1-1